parser/transcript/src/main/java/de/danoeh/antennapod/parser/transcript/PodcastIndexTranscriptParser.java

```diff
@@ -0,0 +1,25 @@
+package de.danoeh.antennapod.parser.transcript;
+
+import org.apache.commons.lang3.StringUtils;
+
+import de.danoeh.antennapod.model.feed.Transcript;
+
+public class PodcastIndexTranscriptParser {
+
+    public static Transcript parse(String str, String type) {
+        if (str == null || StringUtils.isBlank(str)) {
+            return null;
+        }
+
+        str = str.replaceAll("\r\n", "\n");
```

Comment:
This is only relevant to the srt parser, isn't it? Then I would move it to the srt parser, so that the class is self-contained and doesn't break if someone forgets to do this before calling the srt parser